### DISCUSSION TITLE
Do AI algorithms for heat pumps consider the parasitic costs of defrosting in humid environments?

### DISCUSSION THREAD

**Discussion by esc-data-science** [alan-turing-institute/ADViCE#92] [Key sector: Built Environment]
Will the AI algorithm consider the parasitic costs of defrosting in humid environments? In NI we have a humid maritime climate and the amount of defrosting of HPs seems quite high.

**Comment by samyoung-dsci** [comment]
If it's using the right data that includes that effect, then it should be able to. Note that there's some useful 2-minute resolution monitoring data for hundreds of heat pumps in UK homes available here: https://github.com/ES-Catapult/electrification_of_heat
